src/DDTrace/Bootstrap.php

```diff
@@ -178,6 +178,13 @@ private static function initRootSpan(Tracer $tracer)
 
             return $result;
         });
+
+        dd_trace('http_response_code', function () use ($span) {
+            $httpStatusCode = func_get_args();
+            if (is_numeric($httpStatusCode)) {
+                $span->setTag(Tag::HTTP_STATUS_CODE, $httpStatusCode);
+            }
+        });
```

Comment:
To answer your question about why this isn't working: `func_get_args` will return an array, and an array is not numeric. You probably meant to get the first arg out of the array; something like:

```php
$args = func_get_args();
if (isset($args[0]) && \is_numeric($args[0])) {
    $span->setTag(Tag::HTTP_STATUS_CODE, $args[0]);
}
```

Reply:
Thanks @morrisonlevi ! I just modified the code, but it's still not getting inside the callback for `http_response_code`. So I tested it with xdebug yesterday, even with the wrong approach, it did not get inside it

Reply:
Thanks, I think @labbati explained it below why it wasn't working, seems to be passing the tests now